airbyte-integrations/connectors/source-chargebee/metadata.yaml

```diff
@@ -10,7 +10,7 @@ data:
   connectorSubtype: api
```

Comment:
The `baseImage` needs to upgraded to `6.33.0` or higher. We have a recent bug fix that affects usage of the `DatetimeBasedCursor`. We should the latest published version